README.md

```diff
@@ -0,0 +1,16 @@
+# Smart_Watch_One
```

Comment:
Don't need to include copyright in Readme, please discuss with Vishnu and take the Readme files.